### ISSUE TITLE
Client crash searching ores in JEI serch with AE mod?

### ISSUE THREAD

**Issue by chunkedelement** [EpicSquid/MysticalWorld#4]
AE2 version: RV6-Stable-3
Mystical World Version: 1.12.2-1.0.1
Forge Version: 1.12.2-14.23.5.2772
When searching for tin or bronze metals in JEI the client will crash to desktop. I tried disabling the metals/ores in the mystical world config but still get the same result. The crash log states its a facade but unsure why.
If you remove/disable the Mystical World mod and do the search again the game is fine. Not sure what is causing this is it a metals conflict possibly or a AE2 problem? Thanks for your help and work reviving this mod btw the mobs are great.
Basically this:
Ingredient Mod: Applied Energistics 2
	Ingredient Info: 1xitem.appliedenergistics2.facade@0 appliedenergistics2:facade nbt:{damage:0,item:"mysticalworld:copper_block"}

[crash-2018-11-06_21.58.48-client.txt](https://github.com/EpicSquid/MysticalWorld/files/2555973/crash-2018-11-06_21.58.48-client.txt)
[crash-2018-11-05_23.09.13-client.txt](https://github.com/EpicSquid/MysticalWorld/files/2555974/crash-2018-11-05_23.09.13-client.txt)

**Comment by EpicSquid** [comment]
Can you post this on the ae2 issue tracker too? I believe this is an issue with how code chicken lib and mystical lib interact...

**Comment by chunkedelement** [comment]
ok sure no problem, thanks for checking into this, Question if ores are enable/disabled in the mystical world.cfg how will that effect ore gen in worlds that have immersive engineering ore gen enabled. we implemented this in our pack but I didn't disable the ores in the mystical world.cfg. Will we have to turn on IE retro ore gen if disabling the ores in mystical worlds mod? Thanks

**Comment by Daenyth** [comment]
It's reported now at https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3810

**Comment by chunkedelement** [comment]
Thanks for you help in checking on this.

**Comment by Martacus** [comment]
FIxed https://github.com/AppliedEnergistics/Applied-Energistics-2/pull/3844
